### ISSUE TITLE
Crash when capturing IOU

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-server#359]
```
I think I have found a new bug with Packet Capture
 
I have two IOU L3 devices attached to a GNS3 built-in l2 switch
Both are running on the GNS3 VM (selected the VM when I started the switch)
Step 1 - Start packet cap on the switch-end of the link -- This is OK
Step 2 - Start packet cap on the IOU end while the other is running-  This resets/kill the IOU router.
 
Let me know if you need an example project
```

**Comment by noplay** [comment]
I confirm starting a packet capture crash IOU

**Comment by grossmj** [comment]
Confirmed there is an issue with iouyap 0.96.0, it crashes with signal 6 (SIGABRT) when starting a packet capture.

iouyap.log contains:

```
/usr/local/bin/iouyap: Parsing NETMAP...
/usr/local/bin/iouyap: --------------
/usr/local/bin/iouyap: Configuring 513:0/0...
/usr/local/bin/iouyap: starting foreign listener
/usr/local/bin/iouyap: --------------
/usr/local/bin/iouyap: Starting IOU UDS listener
/usr/local/bin/iouyap: Starting IOU UDP listener
/usr/local/bin/iouyap: Main thread going to sleep
/usr/local/bin/iouyap: Received signal 1
/usr/local/bin/iouyap: Stopping listeners and cleaning up
/usr/local/bin/iouyap: Reloading configuration because of SIGHUP
*** Error in `/usr/local/bin/iouyap': free(): invalid next size (fast): 0x0000000002067050 ***
```

Looks like this is a regression introduced in iouyap since version 0.9.2.

**Comment by grossmj** [comment]
Confirmed, problem comes from embedding iniparser in https://github.com/GNS3/iouyap/commit/fc145e3930071de7c13380abdbe2a6fe09234dc1

**Comment by noplay** [comment]
Fixed in iouyap 0.97

**Comment by noplay** [comment]
I embed iniparser 3.2
